{"name": "Verify login with invalid credentials", "status": "passed", "start": 1751978512549, "stop": 1751978514173, "uuid": "3a0bf3e3-69dc-4151-b141-6f2cc1d6f6ea", "historyId": "0b4a530345edabbd36260c6e7a3a603d", "testCaseId": "0b4a530345edabbd36260c6e7a3a603d", "fullName": "tests.api.api_test#test_login_invalid_api", "labels": [{"name": "suite", "value": "API Tests"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "host", "value": "WS01797"}, {"name": "thread", "value": "32112-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.api_test"}]}